schema/index.graphql

```diff
@@ -127,9 +203,43 @@ type Error {
 Shop specific settings and configuration
 """
 type Shop {
+  currencies: [Currency!]!
+  locales: [Locale!]!
```

Comment:
Cases:

1. Shop with for one country (e.g.: germany) with one language will have `locales: [{isoCode: 'de_DE'}]` & `defaults.locale: {isoCode: 'de_DE'}`
2. Shop with one country and multi-language (e.g.: swiss, german & franch): locales: [{isoCode: 'de_CH'}, {isoCode: `fr_CH`}]` & `defaults.locale: {isoCode: 'de_CH'}`
3. Shop with different countries & different languages (e.g.: DACH region): 

```
locales: [{
   isoCode: 'de_DE'
}, {
  isoCode: 'de_CH'
}, {
  isoCode: 'fr_CH'
}, {
  isoCode: 'de_AT'
}],
defaults: {
  locale: {
    isoCode: 'de_DE'
  }
}
```

Reply:
The case for locales instead of just countries is that they transport a lot more general information, like formatting of numbers, dates & times, that are relevant for building localizable frontends.